lib/image_encoder.ex

```diff
@@ -0,0 +1,93 @@
+defmodule PDF417.PNGEncoder do
```

Comment:
Can we rename this file to `png_encoder.ex`? I was looking for this and took me a hot minute to find it 😆

Reply:
weird, I thought I did that. that's what I get for working in the wee hours of the morning.